### ISSUE TITLE
crowdin it translation

### ISSUE THREAD

**Issue by urfausto** [uBlockOrigin/uBlock-issues#2026]
### Prerequisites

- [X] I verified that this is not a filter issue (MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues))
- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### Description

hi, i'm posting this to just gather attention on my request to proofread the translation of my native language. i've already sent a message lot of time ago but no response ever came through and it seems there is no other way to contact an admin. my username is urfausto there.

@gorhill

### A specific URL where the issue occurs

https://crowdin.com/project/ublock/it

### Steps to Reproduce

null

### Expected behavior

null

### Actual behavior

null

### uBlock Origin version

null

### Browser name and version

null

### Operating System and version

null

**Comment by u-RraaLL** [comment]
You should have started a discussion instead.

What's the problem btw? You haven't really explained it.

From what I understand anybody can propose changes and once others vote for these changes, gorhill will pull them with the next release.

**Comment by hirorpt** [comment]
No activity, No trust. 

The lack of proofreaders is indeed a problem.  I sent a message to gorhill requesting the permission after contributing to uBlock for a few weeks on Crowdin, and he immediately grant me. You could do the same thing.


![Screenshot 2022-03-04 at 16-38-13 uBlock Project Activity Stream](https://user-images.githubusercontent.com/82331005/156720899-092862db-c980-45e2-bbf1-7bdc40838728.png)
.

**Comment by gorhill** [comment]
> No activity

Yes, usually I like to see a track record of contributions before granting proofreader status. There are already active contributors for Italian language, so they are at this point preferred candidates for proofreader status, but none of them requested -- but I wouldn't want to alienate them by disregarding their past contributions and granting proofreader to someone who could undo their work. Best is to first start to contribute and collaborate with existing translators.

**Comment by urfausto** [comment]
> The lack of proofreaders is indeed a problem. I sent a message to gorhill requesting the permission after contributing to uBlock for a few weeks on Crowdin, and he immediately grant me. You could do the same thing.

ok thanks. now i'm really busy with real life survival problems. i have also some more project i should work upon and will not do maybe for months or years. my situation is now pretty dramatic but i will try to do what you are saying when i'm more cool.

**Comment by hirorpt** [comment]
@gdonisi @quirico @Mannivu 

What do you think about @urfausto's implying that the quality of uBO's Italian translations are not very good?
Since you seem to be uBO active contributors for Italian, I think your opinion is worthwhile.

**Comment by gdonisi** [comment]
@hirorpt I don't think my translations are wrong (and neither are @quirico and @Mannivu). @urfausto what are the strings that you think are problematic? Anyway from what I understand you just want to be a proofreader, but there are no wrong strings (only some of them are outdated like "I'm an advanced user", which kept the wiki link and like it was said before: large number of votes for the old translation will discourage translators from following changes)

**Comment by urfausto** [comment]
@gdonisi most of the open-crowded software i stumbled upon had a bad to very bad translation lol. plus i've spent so many hours in perfectioning translations from english for so many softwares that i always find something to correct. i'm quite experienced already. there are exceptions though. didn't check ublock in detail but my workflow is simply to get the proofreading and make the entire translation, just the same i did with other softwares.
